src/neo/Network/P2P/Helper.cs

```diff
@@ -16,5 +17,20 @@ public static byte[] GetHashData(this IVerifiable verifiable)
                 return ms.ToArray();
             }
         }
+
+        public static UInt256 XorUInt256List(this UInt256[] values)
+        {
+            if (values == null || values.Length == 0) return UInt256.Zero;
```

Comment:
if length == 1 return values[0] ?

Reply:
Yes, if there is only one hash we can use itself